api-specificatie/openapi.yaml

```diff
@@ -88,7 +88,7 @@ paths:
       tags:
         - Adres
       summary: "levert adressen"
-      description: "Vind een actueel adres met de identificatie van het zoekresultaat uit de operatie get /adressen/zoek, de pandidentificatie of de adresseerbaarobjectidentificatie. Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/fields.feature). Gebruik de expand parameter als je het antwoord wil uitbreiden met (delen van) de gerelateerde resources nummeraanduiding, woonplaats en openbare ruimte, zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/expand.feature)."
+      description: "Vind een actueel adres met de identificatie van het zoekresultaat uit de operatie get /adressen/zoek, de pandidentificatie of de adresseerbaarobjectidentificatie. Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/fields.feature). Als met de field parameter niet is aangegeven dat huisletter of huisnummertoevoeging gewenst is, maar deze zijn wel beschikbaar in de LV BAG, dan worden deze velden teruggegeven in de response, omdat ze één consistent geheel vormen met het huisnummer. Gebruik de expand parameter als je het antwoord wil uitbreiden met (delen van) de gerelateerde resources nummeraanduiding, woonplaats en openbare ruimte, zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/expand.feature)."
```

Comment:
> Als met de field parameter niet is aangegeven dat huisletter of huisnummertoevoeging gewenst is, maar deze zijn wel beschikbaar in de LV BAG, dan worden deze velden teruggegeven in de response, omdat ze één consistent geheel vormen met het huisnummer.

- field parameter --> fields parameter
- zoals het er nu staat is het denk ik niet juist, want wanneer ook huisnummer niet gevraagd wordt, moeten huisletter en huisnummerToevoeging niet geleverd worden.
- zijn wel beschikbaar in de LV BAG --> hebben een waarde